gui/constants.go

```diff
@@ -88,3 +92,69 @@ var (
 		},
 	}
 )
+
+func init() {
+	dirname, err := os.UserHomeDir()
+	if err != nil {
+		log.Fatalf("HOME directory couldn't get")
+	}
+	path := dirname + "/.glsrc"
```

Comment:
we can store the name of the configuration file as a Go constant somewhere else such as `const ConfigurationFile = ".glsrc"`

Reply:
Done in [49682e4](https://github.com/ozansz/gls/pull/25/commits/49682e47904096cefd17374f90e3c0626df095a0).